src/components/Nav/Nav.tsx

```diff
@@ -73,7 +74,7 @@ export class Nav extends React.Component<INavProps, INavState> implements INav {
     }
 
     return (
-      <li key={ linkIndex }>
+      <li role='listitem' key={ linkIndex }>
```

Comment:
I think inside <li> there is <a> that would have aria-label for the actual item link.

From: antonlabunets [mailto:notifications@github.com]
Sent: Monday, August 29, 2016 6:21 PM
To: OfficeDev/office-ui-fabric-react <office-ui-fabric-react@noreply.github.com>
Cc: Ina Teegan <inateeg@exchange.microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [OfficeDev/office-ui-fabric-react] Nav component need to have aria label (#199)


In src/components/Nav/Nav.tsx<https://github.com/OfficeDev/office-ui-fabric-react/pull/199#discussion_r76717157>:

> @@ -73,7 +74,7 @@ export class Nav extends React.Component<INavProps, INavState> implements INav {

>      }

>

>      return (

> -      <li key={ linkIndex }>

> +      <li role='listitem' key={ linkIndex }>

Do you not need an aria label here?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/OfficeDev/office-ui-fabric-react/pull/199/files/989c4d104b9f09f0340852bbac50cf263f94b223#r76717157>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AT1rLop3PzuDhyuaFWZbiURvGlT0E4o-ks5qk4VwgaJpZM4Jsltr>.